compiler/codegen/symbol.cpp

```diff
@@ -786,6 +786,24 @@ static Type* getArgSymbolCodegenType(ArgSymbol* arg) {
   return useType;
 }
 
+// Alter the C code type to avoid _ref_, c_ptr_ prefixes and instead use a
+// type that C understands without additional information (e.g. instead of
+// _ref_int64_t, just int64_t *).  But only for exported symbols like the
+// return type of exported functions, or arguments of those functions.
+static std::string
+transformTypeForPointer(Type* type, bool exported) {
+  std::string typeName = type->codegen().c;
```

Comment:
Shouldn't this be checking for c_ptr with FLAG_DATA_CLASS and for ref with FLAG_REF on type->symbol? The string begins-with stuff is pretty gross.

Reply:
I didn't realize I could use FLAG_DATA_CLASS to detect c_ptrs, thanks!  Is FLAG_REF usable for both the explicit ref types and the qualified ref stuff?

Reply:
For qualified refs the type here probably ends up as a _ref type (that's what getArgSymbolCodegenType does).

Reply:
I actually want FLAG_C_PTR_CLASS, FLAG_DATA_CLASS also grabs _ddata and I *don't* want that